Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -592,7 +592,7 @@
 
 - type: cargoBounty
   id: BountyRollerskates
-  reward: 6500
+  reward: 600
```

Comment:
The only way to get roller skates is the toybox, which costs 900 for 1 pair of skates.

Reply:
> The only way to get roller skates is the toybox, which costs 900 for 1 pair of skates.

You can find it in maintenance lockers

Reply:
skates are relatively rare loot and this one requires two skates for completion.
This bounty will get skipped most of the time unless cargo already has these skates on hand.